dev/ci/ci-vst.sh

```diff
@@ -9,7 +9,7 @@ git_download vst
 
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
-export COMPCERT=bundled
+export COMPCERT=bundled_new FLOCQ=bundled
```

Comment:
```suggestion
export COMPCERT=bundled_new
export FLOCQ=bundled
```
is nicer IMO

Reply:
also it seems it didn't work
>Warning: Cannot open compcert_new/export

and then some error

Reply:
Indeed, doesn't seem to work, I'm waiting for the discussion there: https://coq.zulipchat.com/#narrow/stream/250632-Coq-Platform-devs-.26-users/topic/coq-native to settle